packages/pwa-kit-dev/src/ssr/server/build-dev-server.test.js

```diff
@@ -304,10 +302,8 @@ describe('DevServer listening on http/https protocol', () => {
                 }
             )
             server = _server
-            return insecureFetch(`${protocol}://localhost:${TEST_PORT}`).then((response) => {
-                expect(response.ok).toBe(true)
-                return Promise.resolve()
-            })
+            const response = await insecureFetch(`${protocol}://localhost:${TEST_PORT}`)
+            expect(response.ok).toBe(true)
```

Comment:
This test is flaky, so this is just an attempt to clean it up and maybe improve the flakiness (but probably not).